### PR TITLE
system-variables.md: update tidb_ddl_enable_fast_reorg

### PR DESCRIPTION
**Pull request by tangenta**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): https://github.com/pingcap/tidb/pull/39267

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -871,8 +871,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 - 作用域：GLOBAL
 - 是否持久化到集群：是
-- 默认值：`OFF`
-- 这个变量是用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。如果本开关启动，TiDB 在进行创建索引任务时将会采用更加高效的方式完成索引创建。
+- 默认值：`ON`
+- 这个变量用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度，对于数据量较大的表有一定的性能提升。
+- 要验证已经完成的 ADD INDEX 操作是否使用了“添加索引加速功能”，可以在 `ADMIN SHOW DDL JOBS` 中查看 `JOB_TYPE` 一列中是否含有 `ingest` 字样。
```
Comment (Oreoxmt) [inline system-variables.md:876]:
```suggestion
- 要验证已经完成的 `ADD INDEX` 操作是否使用了添加索引加速功能，可以在 `ADMIN SHOW DDL JOBS` 中查看 `JOB_TYPE` 一列中是否含有 `ingest` 字样。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: system-variables.md [hunk 2]
```diff
@@ -871,8 +871,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 - 作用域：GLOBAL
 - 是否持久化到集群：是
-- 默认值：`OFF`
-- 这个变量是用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。如果本开关启动，TiDB 在进行创建索引任务时将会采用更加高效的方式完成索引创建。
+- 默认值：`ON`
+- 这个变量用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度，对于数据量较大的表有一定的性能提升。
+- 要验证已经完成的 `ADD INDEX` 操作是否使用了添加索引加速功能，可以在 `ADMIN SHOW DDL JOBS` 中查看 `JOB_TYPE` 一列中是否含有 `ingest` 字样。
```
Comment (Oreoxmt) [inline system-variables.md:876]:
```suggestion
- 要验证已经完成的 `ADD INDEX` 操作是否使用了添加索引加速功能，可以执行 [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md#admin-show-ddl-jobs) 中查看 `JOB_TYPE` 一列中是否含有 `ingest` 字样。
```

**Review by Benjamin2037 — APPROVED** [review]
LGTM

File: system-variables.md [hunk 3]
```diff
@@ -871,8 +871,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 - 作用域：GLOBAL
 - 是否持久化到集群：是
-- 默认值：`OFF`
-- 这个变量是用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。如果本开关启动，TiDB 在进行创建索引任务时将会采用更加高效的方式完成索引创建。
+- 默认值：`ON`
+- 这个变量用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度，对于数据量较大的表有一定的性能提升。
```
Comment (Oreoxmt) [inline system-variables.md:875]:
```suggestion
- 这个变量用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。开启该变量对于数据量较大的表有一定的性能提升。
```

**Review by Oreoxmt — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: system-variables.md [hunk 4]
```diff
@@ -871,13 +871,12 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 - 作用域：GLOBAL
 - 是否持久化到集群：是
-- 默认值：`OFF`
-- 这个变量是用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。如果本开关启动，TiDB 在进行创建索引任务时将会采用更加高效的方式完成索引创建。
+- 默认值：`ON`
+- 这个变量用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。开启该变量对于数据量较大的表有一定的性能提升。
```
Comment (TomShawn) [inline system-variables.md:875]: 「数据量较大」是多大呢？

File: system-variables.md [hunk 5]
```diff
@@ -871,13 +871,12 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+- 要验证已经完成的 `ADD INDEX` 操作是否使用了添加索引加速功能，可以执行 [`ADMIN SHOW DDL JOBS`](/sql-statements/sql-statement-admin-show-ddl.md#admin-show-ddl-jobs) 语句查看 `JOB_TYPE` 一列中是否含有 `ingest` 字样。
```
Comment (TomShawn) [inline system-variables.md:876]: 是否要在 https://docs.pingcap.com/zh/tidb/dev/sql-statement-add-index 这篇文档里也提一下索引加速功能呢？

File: system-variables.md (hunk 4 above)
Comment (tangenta) [reply system-variables.md:875]: 一百万行以上吧，不过没法定一个具体值说超过这个值"一定"有较大提升，这个取决于表的宽度、索引列大小、region 数量等等...

File: system-variables.md (hunk 5 above)
Comment (tangenta) [reply system-variables.md:876]: 应该不需要，v6.5 开始已经默认打开了开关。后续会针对这个功能写一些原理介绍的文章。

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**